scripts/run-all.sh

```
@@ -0,0 +1,168 @@
+for prefix in "${!accounts[@]}"; do
+  account_id="${accounts[$prefix]}"
+  
+  if [[ -z "$account_id" ]]; then
+    echo "⚠️  Ghostfolio Account ID for '$prefix' not set in .env. Skipping."
+    continue
+  fi
+
+  mkdir -p "out/${prefix}"
+
+  # Find all CSV files for this prefix in the input directory
+  mapfile -t csv_files < <(ls -1 input/${prefix}*.csv 2>/dev/null | sort)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)